tst/testbugfix/2013-06-14-t00300.tst

```diff
@@ -3,7 +3,7 @@ gap> foo:=function() return 42; end;
 function(  ) ... end
 gap> DeclareObsoleteSynonym("bar","foo");
 gap> oldLevel := InfoLevel(InfoObsolete);;
-gap> SetInfoLevel(InfoObsolete,1);
+gap> SetInfoLevel(InfoObsolete,2);
```

Comment:
Non-blocker, but: it'd be nice if the new features were also tested. Like the fact that info messages are now printed only once; or the new `level_arg`